lib/iris/analysis/_grid_angles.py

```diff
@@ -65,13 +66,13 @@ def _latlon_from_xyz(xyz):
     Returns:
 
         lonlat : (array)
-            spherical angles, of dims (2, <others>), in radians.
+            spherical angles, of dims (2, <others>), in degrees.
```

Comment:
I do prefer that this is working in degrees to be consistent with the `_3d_xyz_from_latlon` function

Reply:
hi @lbdreyer 
I'll look into these now if you like, 
but in principle I had already deferred these "comments on the comments" to a follow-on : #3109, also linked from #3114.

I have been working on the follow-up, to address those remaining issues + add missing tests ...